src/apps/api/serializers/datasets.py

```diff
@@ -59,9 +60,13 @@ class Meta:
             'is_public',
             'key',
             'was_created_by_competition',
-            'in_use'
+            'in_use',
+            'file_size',
         )
 
+    def get_file_size(self, instance):
+        return instance.data_file.size
```

Comment:
you could make this a `serializers.IntegerField(source="instance.data_file.size", read_only=True)` I think?

Reply:
(fine as is, though!)

Reply:
added this as a field on the data model.